### ISSUE TITLE
Layers to be finetuned

### ISSUE THREAD

**Issue by FlyHighest** [Yuanshi9815/OminiControl#46]
I notice that in your released subject omini loras, only ``attn.to_k``, ``attn.to_q`` are used. Why not include attn.to_v?

I train my own omini subject lora with all ``to_k, to_q, to_v`` on my own dataset and find that in the early stage, the generated images  are significantly downgraded, and after thousands of steps, the model generate identical images with input condition images. What's the reason? Is it because ``to_v`` is included in lora training? 

Thanks again for your work. Wish you best luck!

**Comment by OranHe** [comment]
Which pipeline did you use to train your own model? Could you share the training code？

**Comment by FlyHighest** [comment]
@OranHe I modify the diffusers example lora training code(https://github.com/huggingface/diffusers/blob/main/examples/dreambooth/train_dreambooth_lora_flux.py), and use ``transformer_forward`` function in this repo to obtain ``model_pred``. Some forward functions need small fixes to support gradient checkpointing.  Also, there are some conflicts with latest diffusers. I'm not sure if my implementation aligns with the author.

**Comment by Yuanshi9815** [comment]
Hi @FlyHighest ! Thanks for your interest, and Merry Christmas! 🎄

The experiments in our paper used the models in the `experimental` folder with full parameter fine-tuning. However, for the models in the `omini` folder, we've switched to a more efficient approach using partial parameter tuning through LoRA.

Regarding your training issue, I can't speak definitively about the specific training issue you encountered, but we'll be releasing our training code soon which may help clarify these questions. 🙌

@OranHe
